### PR TITLE
LPS-117228 Create tests

### PR DESCRIPTION
**Pull request by alinedoleron**
Issue: https://issues.liferay.com/browse/LPS-117228

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+		const {getByText} = render(createCardShortcut());
+
+		const field1 = getByText('Field 1');
+		const field2 = getByText('Field 2');
+
+		expect(field1.closest('.selected')).toBe(null);
+		expect(field2.closest('.selected')).not.toBe(null);
+	});
+});
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js:107]: @alinedoleron, why did you remove the last test added on the last pull which used the `scrollIntoViewMock`?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	{
+		label: 'Field 1',
+		name: 'field1',
+		type: 'text',
+	},
+	{
+		label: 'Field 2',
+		name: 'field2',
+		type: 'radio',
+	},
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js:31]: @alinedoleron, the `options` attribute should be removed only for the Text field and not for the Radio, after all a Radio field has options.

**Comment by natocesarrego** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js (hunk 1 above)
Comment (alinedoleron) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js:107]: Hi @natocesarrego, I removed it because on CardShortcut component I'm not doing srollIntoView call anymore. I've made some changes after you say to do the lazy-initial-state, so the call is not necessary.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js (hunk 2 above)
Comment (alinedoleron) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js:31]: Hi @natocesarrego, I can do that. But I removed since on this tests the options are not being used.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js (hunk 1 above)
Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js:107]: Great! Thanks for the clarification.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js (hunk 2 above)
Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js:31]: Ok, @alinedoleron. I'll forward your pull. 🙂

**Comment by natocesarrego** [comment]
ci:forward
